src/base/http/httputils.cpp

```diff
@@ -27,42 +27,60 @@
  * exception statement from your version.
  */
 
-#include "responsegenerator.h"
+#include "httputils.h"
```

Comment:
look at the file name: src/base/http/httputils.cpp
httputils.cpp in http folder, isn't it redundant?

Reply:
Since we use QMake we can't have sources having same name even in different folders since it produces object files in the same folder.